client/src/form/AddNewListBar.js

```diff
@@ -50,48 +62,49 @@ export default function BasicTextFields(props) {
   return (
     <form noValidate autoComplete="off" onSubmit={handleSubmit}>
 
-        <Grid container spacing={1} justify="center" align="center">
+        <Grid container spacing={1} justify="center" align="center" style={{backgroundColor: '#fbfcff'}}>
             <Grid item xs={10} md={8} style={{textAlign:'center'}}>
-                <TextField 
-                    fullWidth
-                    label="Title"
+            <Typography component="h5" style={{fontWeight: 'bold'}}>
+                Add a title *
+            </Typography>
+            <Box className={classes.box} borderRadius={50} flexGrow={1}>
+                <InputBase 
```

Comment:
Any reason why InputBase instead of TextField?

Reply:
With TextField, we have three variants, standard (horizontal line), filled (bkg colour) and outlined (square bordered box) InputBase has no design and just provides basic text. Since I have designed a Box outside the text inputs, I thought of using that. Can switch back to TextField if we want!